readme.md

```diff
@@ -83,6 +83,24 @@ Default: `true`
 
 Overwrite existing files.
 
+##### trimPathComponents
+
+Type: `number`\
+Default: `0`
+
+Omits the specified amount of parent directories from files being copied.
+
+```js
+import cpy from 'cpy';
+
+await cpy('src/**/*.js', 'destination/flatSourceFiles', {
+	trimPathComponents: 2
+});
+```
+
+Where the destination paths of `*.js` files within subdirectories of the `src` directory
+will be flattened to a reduced structure another `destination/flatSourceFiles` path.
```

Comment:
Don't hard-wrap.

This is missing from index.d.ts

And I think it needs a rewrite. I don't really understand what it's trying to say.